src/background.ts

```diff
@@ -31,7 +31,10 @@ const storageManager = initStorex()
 const notifications = new NotificationBackground({ storageManager })
 notifications.setupRemoteFunctions()
 
-const directLinking = new DirectLinkingBackground({ storageManager, getDb })
+export const directLinking = new DirectLinkingBackground({
```

Comment:
@ShishKabab what do you think about these? They are imported in the `imports/background/item-processor` module to call certain DB ops. 

When we wrote the current version of imports feature, we hadn't reached the point where we had a standard layout for background script feature modules and they interact with the DB, etc. So certain modules import and use functions directly from `src/search` (see rough layout: https://github.com/WorldBrain/Memex/blob/develop/src/imports/imports-arch.png). So in that sense, I can understand why we might just do it like this for now until we rewrite at least the main interface to the imports feature to be like our newer features.